src/aap_eda/api/filters/organization.py

```diff
@@ -0,0 +1,34 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import django_filters
+
+from aap_eda.core.models import Organization
+
+
+class OrganizationFilter(django_filters.FilterSet):
+    name = django_filters.CharFilter(
```

Comment:
@Dostonbek1 Should these be coming from DAB?

Reply:
@mkanoor I think that would be ideal. I know we've discussed using the filters from DAB for other views as well. So I wonder if this should be done in a separate epic for all views later on or use it for this view then implement for others later. Thoughts?